src/network/networkd-network.c

```diff
@@ -687,6 +687,8 @@ static Network *network_free(Network *network) {
         free(network->dhcp6_mudurl);
         strv_free(network->dhcp6_user_class);
         strv_free(network->dhcp6_vendor_class);
+        set_free(network->dhcp_netlabels);
```

Comment:
Ah, following the previous change, maybe this should be `set_free_free(...)` as well in this and the following three cases?

Reply:
Thanks, nice catch!